bftengine/src/bftengine/ReplicaBase.hpp

```diff
@@ -103,6 +103,8 @@ class ReplicaBase {
 
   //////////////////////////////////////////////////
   // METRICS
+  std::chrono::seconds last_dump_time_;
```

Comment:
Like before: please add metrics context to the dama members' names.

Reply:
Done